packages/opds-web-client/src/components/context/__tests__/StoreContextTest.tsx

```diff
@@ -0,0 +1,90 @@
+import * as React from "react";
+import { expect } from "chai";
+import { stub } from "sinon";
+
+import * as PropTypes from "prop-types";
+import { shallow, mount } from "enzyme";
+import { ReactReduxContext } from "react-redux";
+
+import buildStore from "../../../store";
+
+import StoreContextProvider from "../StoreContext";
+import PathForProvider from "../PathForContext";
+import { PathFor } from "../../../interfaces";
+import { isNull } from "util";
+// import { mockRouterContext } from "../../../routing";
+
+/**
+ *  Testing the StoreContext component. Based on old OPDSCatalog Tests
+ *    - if not given state, it will build state on its own
+ *      (seems like this should be a test of buildState maybe?)
```

Comment:
I think you meant `buildStore`? I don't think there are any tests for it right now.

Reply:
I did mean that. And sounds good, this test suite will at least ensure a store gets built when one isn't provided.